view/src/ResultViewer.js

```diff
@@ -0,0 +1,121 @@
+// Component for visualisation of found semantic differences.
+// Author: Lukas Petr
+
+import { useEffect, useState } from "react";
+import { load } from "js-yaml";
+
+import Navbar from "react-bootstrap/Navbar";
+import Container from "react-bootstrap/Container";
+import ResultNavigation from "./components/ResultNavigation";
+import FunctionListing from "./components/FunctionListing";
+import Difference from "./components/Difference";
+
+import Result from "./Result.js";
+
+const DEFAULT_DESCRIPTION_FILE = "diffkemp-out.yaml";
+
+/**
+ * Returns name of the last directory in path.
+ */
+function getLastFolderName(path) {
```

Comment:
I found out that since webpack v5 React does not include polyfills for node.js core modules (eg path) by default. I used `path-browserify` which is an implementation of the Node.js `path` module for browsers.